api/v1alpha1/endpointservice_types.go

```diff
@@ -51,6 +51,10 @@ type AWSEndpointServiceStatus struct {
 	// +optional
 	DNSName string `json:"dnsName,omitempty"`
 
+	// DNSName are the names for the records created in the hypershift private zone
```

Comment:
how is this different from the above field?

Reply:
It is different in that it can hold more than one value

Reply:
so the semantic is the same, I don't we need two different fields to represent 1 vs multiple. Can we drop DNSName and keep only []DNSName where you can have 1..N?

Reply:
Yeah, the semantic is the same. We can deprecate the field, but we have to keep it around for or as long as we support an OCP version where the HCP controller does not have this change